docs/vanilla.md

```diff
@@ -0,0 +1,144 @@
+    * Please make sure port **902** is opened on each ESXi host based on the requirement of the VDDK NBD transport mode.
+
+## Install
+1. [Install Velero](https://velero.io/docs/main/basic-install/)
+2. [Install Object Storage Plugin](#install-object-storage-plugin)
+3. [Install Velero Plugin for vSphere](#install-velero-plugin-for-vsphere)
+
+### Install Object Storage Plugin
+Volume backups are stored in an object store bucket, e.g., S3. Currently these are stored in the same bucket configured for
+the object storage plugin of Velero. Before installing the vSphere plugin, please install and configure the object storage plugin.
+Here is an example, [velero-plugin-for-aws](https://github.com/vmware-tanzu/velero-plugin-for-aws/blob/master/README.md).
+
+
+### Install Velero Plugin for vSphere
+```bash
+velero plugin add <plugin-image>
+```
+
+For Version 1.1.0 the command is
+
+```bash
+velero plugin add vsphereveleroplugin/velero-plugin-for-vsphere:1.1.0
+```
+
+## Uninstall
+To uninstall the plugin, run the following command to remove the InitContainer of velero-plugin-for-vsphere from the Velero deployment first.
+```bash
+velero plugin remove <plugin image>
+```
+To finish the cleanup, delete the Backup Driver deployment, Data Manager daemonset and their related CRDs.
+```bash
+kubectl -n velero delete deployment.apps/backup-driver
+kubectl delete crds backuprepositories.backupdriver.cnsdp.vmware.com \
+                    backuprepositoryclaims.backupdriver.cnsdp.vmware.com \
+                    clonefromsnapshots.backupdriver.cnsdp.vmware.com \
+                    deletesnapshots.backupdriver.cnsdp.vmware.com \
+                    snapshots.backupdriver.cnsdp.vmware.com
+kubectl -n velero delete daemonset.apps/datamgr-for-vsphere-plugin
+kubectl delete crds uploads.datamover.cnsdp.vmware.com downloads.datamover.cnsdp.vmware.com
+```
+
+## Backup
+1. [Backup vSphere CNS Block Volumes](#backup-vsphere-cns-block-volumes)
+2. [Backup vSphere CNS File Volumes](#backup-vsphere-cns-file-volumes)
+### Backup vSphere CNS Block Volumes
+To use the plugin, run a Velero backup and specify the --snapshot-volumes flag.
+
+```bash
+velero backup create my-backup --include-namespaces=my-namespace --snapshot-volumes
+```
+
+Your backup will complete after the local snapshots have completed and your Kubernetes metadata has been uploaded to the object
+store specified.  At this point, all of the data may not have been uploaded to your S3 object store.  Data movement happens in the
+background and may take a significant amount of time to complete.
+
+#### S3 data
+Your volume data is stored in the Velero bucket with prefixes beginning with *plugins/vsphere-astrolabe-repo*
+Velero versions prior to 1.3.2 will fail when using a bucket that has objects with the *plugins* prefix.  If you have
+multiple Velero instances sharing a common bucket, please be sure to upgrade all of the to 1.3.2 before making any
+backups with the vSphere plugin
```

Comment:
Change to 1.5.1 as this page is about plugin 1.1

Reply:
I don't think we need the section, `S3 Data`, any more, as it was applied to pre-v1.3.2 Velero. As we are going to require Velero v1.5.1 or above for this release, we don't need to mention it anymore. What do you think? @xing-yang

Reply:
Sounds good.